website/content/docs/v0.14/Learn More/architecture.md

```diff
@@ -18,6 +18,17 @@ This imposes a clear separation of concerns and ensures that changes over time w
 The benefit is that each component may be iterated and changed as its needs dictate, so long as the external API is controlled.
 This is a key component in reducing coupling and maintaining modularity.
 
+## File system partitions
+
+Talos uses these partions with the following labels:
+
+    - **EFI**: The EFI partition, which is the first partition on the disk and used for EFI boot.
```

Comment:
should we just use numbered list here to avoid `the first`, `the second`, etc. ?